tests/unit/mathematics/linalg/Viennacl_backend_unittest.cc

```diff
@@ -0,0 +1,55 @@
+#include <shogun/lib/config.h>
```

Comment:
why not call the file `LinalgBackendViennaCL_unittest.cc` just like all other tests are named